helm/.gitignore

```
@@ -2,3 +2,4 @@
 chart/templates/rendered.yaml
 chart/templates/crds.yaml
 chart/crds/*.yaml
+/chart/templates/*.yaml
```

Let's remove lines 2 and 3 then ;) 
```suggestion
chart/templates/*.yaml

```